### DISCUSSION TITLE
How to get the CGA “low” palette

### DISCUSSION THREAD

**Discussion by bronger** [robhagemans/pcbasic#232] [Q&A] [answered]
Apparently formerly, PC-BASIC offered a `--cga-low=True` command line option. Is there a current way to accomplish the same thing? I see yellow in the `SCREEN 1:COLOR 1,0` palette, however, I would prefer brown.

**Comment by robhagemans** [comment, marked as answer]
The BASIC command `out &h3d9,0` will select the low-intensity alternative palette
